.github/mergify.yml

```diff
@@ -18,6 +18,7 @@ pull_request_rules:
       - base=unstable
       - label=trivial
       - author=@sigp/lighthouse
+      - conflict
```

Comment:
should this be `conflict=false`?

Reply:
Hi Jimmy, thanks for noticing it forgot to update, `= false` doesn't work for boolean types:
`Operators are invalid for Boolean attribute: `conflict``
I used the `-` operator to negate it, see [here](https://docs.mergify.com/configuration/conditions/#modifiers)